src/explore-education-statistics-common/src/services/dataBlockService.ts

```diff
@@ -129,6 +129,8 @@ export interface DataBlockRerequest {
 
 export interface DataBlock {
```

Comment:
I think the optional additions here are mutating DataBlock into something it shouldn't be, I wouldn't expect a single datablock outside of a contentblock context to have an order - maybe needs rethinking??

Perhaps something like `ContentBlock<DataBlock>` needs to exist?